examples/basic/chat-search.py

```diff
@@ -88,13 +90,20 @@ def chat(opts: CLIOptions) -> None:
         vecdb=None,
     )
     agent = ChatAgent(config)
-    agent.enable_message(GoogleSearchTool)
+
+    if opts.provider == "google":
+        agent.enable_message(GoogleSearchTool)
+    elif opts.provider == "sciphi":
+        agent.enable_message(SciPhiSearchRAGTool)
+    else:
+        raise ValueError(f"Unsupported search provider {opts.provider} specified.")
+
     task = Task(
         agent,
         system_message="""
         You are a helpful assistant. You will try your best to answer my questions.
         If you cannot answer from your own knowledge, you can use up to 5 
-        results from the `web_search` tool/function-call to help you with 
+        results from the `web_search` or `web_search_rag` tool/function-call to help you with 
```

Comment:
In the sci-phi tool you can just set `request = "web_search"`, so we don't need to change this line. In general in a ToolMessage, the `request` param is used to specify which ChatAgent method will handle the tool. But since this tool stateless, this name actually is immaterial (Langroid attaches the `handle` method to the ChatAgent, under the hood).

Reply:
Actually ignore the part about changing the sys msg, I see it doesn't need to change.

Reply:
sorry, wdym? Happy to make the mod, just a bit confused

Reply:
I meant that you can set the “request” param in your tool to be same as for google search , “web_search”. This way you don’t need to change the sys msg since it will map to the right tool based on the user choice. 

Ignore the other parts of what I said, sorry about the confusion 😀